frontend/package.json

```diff
@@ -8,7 +8,9 @@
     "test:unit": "vue-cli-service test:unit",
     "test:e2e": "testcafe chrome tests/e2e",
     "lint": "vue-cli-service lint",
-    "test": "npm run test:unit"
+    "test": "npm run test:unit",
+    "format:check": "prettier -c --loglevel debug --config-precedence prefer-file ./src",
```

Comment:
Maybe we can remove loglevel now that our problem is fixed, it will generate a more clean input of non formatted files